update.py

```diff
@@ -1 +1 @@
-print*"hello")
+print("hello")
```

Comment:
Reviewing changes in **update.py**:

The correction made to the print statement makes it functional and syntactically correct.